### ISSUE TITLE
No Implicit Conversion Bug

### ISSUE THREAD

**Issue by bgeltz** [mustache/mustache#234]
v1.0.4 breaks compilation with ronn.  I'm on Ubuntu 14.04.  The following commands:

```
mkdir tmp
cd tmp
mkdir -p ruby/bin
gem install ronn -i ruby
export GEM_PATH=ruby
touch blank.ronn 
./ruby/bin/ronn blank.ronn
```

will produce the following error message:

```
$ ./ruby/bin/ronn blank.ronn
     roff: ./blank                                    
     html: ./blank.html                                          +man
/tmp/tmp/ruby/gems/mustache-1.0.4/lib/mustache.rb:282:in `merge': no implicit conversion of nil into Hash (TypeError)
        from /tmp/tmp/ruby/gems/mustache-1.0.4/lib/mustache.rb:282:in `templateify'
        from /tmp/tmp/ruby/gems/mustache-1.0.4/lib/mustache.rb:122:in `render'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/lib/ronn/template.rb:14:in `render'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/lib/ronn/document.rb:244:in `to_html'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/lib/ronn/document.rb:221:in `convert'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/bin/ronn:191:in `block (2 levels) in <top (required)>'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/bin/ronn:181:in `each'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/bin/ronn:181:in `block in <top (required)>'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/bin/ronn:167:in `each'
        from /tmp/tmp/ruby/gems/ronn-0.7.3/bin/ronn:167:in `<top (required)>'
        from ./ruby/bin/ronn:23:in `load'
        from ./ruby/bin/ronn:23:in `<main>'
```

**Comment by locks** [comment]
Hi @bgeltz, thank you for reporting this problem!

I have yanked 1.0.4 due to a breaking change that sneaked in. This should be fixed in 1.0.5, please re-open if the problem persists.

**Comment by bgeltz** [comment]
Thanks!  v1.0.5 seems to resolve the issue on my end.
